components/viz/client/client_shared_bitmap_manager.cc

```diff
@@ -186,10 +190,12 @@ void ClientSharedBitmapManager::NotifyRasterizedSharedBitmap(
     size_t memory_size,
     void* memory,
     SharedBitmapId id) {
-  uint8_t* pixels = static_cast<uint8_t*>(memory);
-  std::vector<uint8_t> pixels_vec(pixels, pixels + memory_size);
-  (*shared_bitmap_allocation_notifier_)
-      ->DidRasterizeSharedBitmap(memory_size, pixels_vec, id);
+  if (base::Castanets::IsEnabled()) {
```

Comment:
NotifyRasterizedSharedBimap function is called if Castanets is enabled.
I think the runtime flag is not necessary in this function.

Reply:
From cc/resources/video_resource_updater.cc => NotifyRasterizedTile is called even though castanets is disabled, so we need this until Media patches are merged.

Reply:
Okay, I'll make a patch for MM and handle this. :)